.gitignore

```
@@ -37,7 +39,6 @@ waf-*
 waf3-*
 *latexmk
 *synctex.gz
-.*
 !.environment.osx.yml
```

```suggestion
.
!.github
```
Please continue to ignore hidden files and whitelist those that should be included under VC.

The whitelist patterns for the environment files are from a very distant past and can all be removed...